apps/sodo-search/src/search-index.js

```diff
@@ -133,4 +137,79 @@ export default class SearchIndex {
             tags: this.#normalizeSearchResult(tags)
         };
     }
+
+    checkLocaleCjk(locale) {
+        if (!locale) {
+            return false;
+        }
+        // Chinese
+        if (locale === 'zh' || locale === 'zh-Hans' || locale === 'zh-Hant') {
+            return true;
+        }
+        // Japanese
+        if (locale === 'ja') {
+            return true;
+        }
+        // Korean
+        if (locale === 'ko') {
+            return true;
+        }
+        return false;
+    }
+
+    getCjkFlexsearchDocumentPatchOptions(locale) {
+        const isLocaleCjk = (() => {
+            if (!locale) {
+                return false;
+            }
+            // Chinese
+            if (locale === 'zh' || locale === 'zh-Hans' || locale === 'zh-Hant') {
+                return true;
+            }
+            // Japanese
+            if (locale === 'ja') {
+                return true;
+            }
+            // Korean
+            if (locale === 'ko') {
+                return true;
+            }
+            return false;
+        })();
```

Comment:
Is there a reason this isn't using the `checkLocaleCjk` function above?

Reply:
Actually, I forgot to delete it.